products/multi_receipts_detector/response_v1/summary_full.rst

```diff
@@ -0,0 +1,25 @@
+########
+Document
+########
+:Mindee ID: d7c5b25f-e0d3-4491-af54-6183afa1aaab
+:Filename: default_sample.jpg
+
+Inference
+#########
+:Product: mindee/multi_receipts_detector v1.0
+:Rotation applied: Yes
+
+Prediction
+==========
+
```

Comment:
Json shows top-level results here. Which one is right?

Reply:
The PR was done before the product was reworked, so probably all the .rst files need to be reworked.